service/lib/agama/storage/finisher.rb

```diff
@@ -332,13 +411,20 @@ def run
 
       # Step to write the mountlist file for Iguana, if needed
       class IguanaStep < Step
+        include Yast::I18n
+
         IGUANA_PATH = "/iguana"
         private_constant :IGUANA_PATH
         IGUANA_MOUNTLIST = File.join(IGUANA_PATH, "mountlist").freeze
         private_constant :IGUANA_MOUNTLIST
 
+        def initialize(logger)
+          super
+          textdomain "agama"
```

Comment:
do we need to have textdomain in each chield? Won't be enough to havve it in Step?

Reply:
I don't know. Would it work if the base class `Step` is moved to a separate file?

Reply:
Anyway, in this case using textdomain in the base class is enough for the script. So let's do it.